packages/components/organisms/f-checkout/src/store/checkout.module.js

```diff
@@ -139,6 +139,37 @@ export default {
             commit('UPDATE_AVAILABLE_FULFILMENT_TIMES', data);
         },
 
+        /**
+         * Get the basket details from the backend and update the state.
+         *
+         * @param {Object} commit - Automatically handled by Vuex to be able to commit mutations.
+         * @param {Object} payload - Parameter with the different configurations for the request.
+         */
+        getBasket: async ({ commit }, {
+            url,
+            tenant,
+            language,
+            timeout
+        }) => {
+            // TODO: deal with exceptions.
+            const config = {
+                method: 'get',
+                headers: {
+                    'Content-Type': 'application/json',
+                    'Accept-Tenant': tenant,
```

Comment:
We had to remove this from other the Checkout API calls. Do we need it for this API?

Reply:
Yep. The basket orchestrator calls in production have it